src/app/AttributeAccessInterface.h

```diff
@@ -414,6 +414,23 @@ class AttributeAccessInterface
      */
     virtual CHIP_ERROR Write(const ConcreteDataAttributePath & aPath, AttributeValueDecoder & aDecoder) { return CHIP_NO_ERROR; }
 
+    /**
+     * Indicates the start a a series of list operations, this function will be called before first Write to the list.
```

Comment:
```suggestion
     * Indicates the start of a series of list operations. This function will be called before first Write to the list.
```
So if the client sends, in the same write request, a write of [1, 2] to a list attribute and then a write of [3, 4] to the same list attribute (that is, two full-list writes), will that call OnListWriteBegin once or twice?

Reply:
Will call OnListWriteBegin once, we will regard them as the same transaction.

<- OnListWriteBegin
<- Write [1,2] (full write)
<- Write [3,4] (full write)
<- OnListWriteEnd